app/routes/_gcn.circulars.$circularId.($version)/route.tsx

```diff
@@ -0,0 +1,68 @@
+/*!
+ * Copyright © 2023 United States Government as represented by the
+ * Administrator of the National Aeronautics and Space Administration.
+ * All Rights Reserved.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+import type { DataFunctionArgs, HeadersFunction } from '@remix-run/node'
+import { json } from '@remix-run/node'
+import { useLoaderData } from '@remix-run/react'
+
+import { get } from '../_gcn.circulars/circulars.server'
+import { PlainTextBody } from './Body'
+import { FrontMatter } from './FrontMatter'
+import { feature, origin } from '~/lib/env.server'
+import {
+  getCanonicalUrlHeaders,
+  pickHeaders,
+  publicStaticShortTermCacheControlHeaders,
+} from '~/lib/headers.server'
+import type { BreadcrumbHandle } from '~/root/Title'
+
+export const handle: BreadcrumbHandle<typeof loader> = {
+  breadcrumb({ data }) {
+    if (data) {
+      const { circularId, subject } = data
+      return `${circularId}: ${subject}`
+    }
+  },
+}
+
+export async function loader({
+  params: { circularId, version },
+}: DataFunctionArgs) {
+  if (!feature('CIRCULAR_VERSIONS') && version)
+    throw new Response(null, { status: 404 })
+  if (!circularId)
+    throw new Response('circularId must be defined', { status: 400 })
```

Comment:
```suggestion
  invariant(circularId)
```

It's a precondition that circularId must be a non-empty string because Remix won't route us to this module otherwise.